packages/open-payments/src/client/index.ts

```diff
@@ -0,0 +1,36 @@
+import { createOpenAPI } from 'openapi'
+
+import { ILPStreamConnection, IncomingPayment } from '../types'
+import config from '../config'
+import { getIncomingPayment } from './incoming-payment'
+import { getILPStreamConnection } from './ilp-stream-connection'
+import { createAxiosInstance, GetArgs } from './requests'
+
+export interface CreateOpenPaymentClientArgs {
+  timeout?: number
+}
+
+export interface OpenPaymentsClient {
+  incomingPayment: {
+    get(args: GetArgs): Promise<IncomingPayment>
+  }
+  ilpStreamConnection: {
+    get(args: GetArgs): Promise<ILPStreamConnection>
+  }
+}
+
+export const createClient = async (
+  args?: CreateOpenPaymentClientArgs
+): Promise<OpenPaymentsClient> => {
+  const axios = createAxiosInstance(args)
+  const openApi = await createOpenAPI(config.OPEN_PAYMENTS_OPEN_API_URL)
```

Comment:
using the same version to validate the Open API as was used for generating the types in the first place